.github/ISSUE_TEMPLATE/bug-chinese.yml

```
@@ -1,75 +1,113 @@
+          required: true
+
   - type: textarea
     id: bug-description
     attributes:
-      label: "Bug 描述"
-      description: "清晰地描述，请提供 hexo 根目录的网站配置文件 `_config.yml` 和主题配置文件 `_config.redefine.yml` 或 `_config.yml` 中有关的项目。如果有敏感信息请打码（替换一些字符"
+      label: Bug 描述
+      description: 请详细描述您遇到的问题，并提供相关配置文件内容（敏感信息请用 *** 替换）
+      placeholder: |
+        问题描述：
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)